### ISSUE TITLE
The  weight trained by my custom data is pretty larger than the  pre-trained weight provided by authors

### ISSUE THREAD

**Issue by Septembit** [SwinTransformer/Swin-Transformer-Object-Detection#111]
Hi, every one, why my weight is so large,  about 3 times than the author provided.
My custom data has 4 classes, which is less than COCO data. But my weight's size (mask-rcnn, swin-tiny) is 569 MB, however, the author's is only 191.5 MB.

**Comment by impiga** [comment]
Hi, each checkpoint file contains not only model weight, but also optimizer states and other meta data. 

We only release the model weight.
